src/utils.js

```diff
@@ -126,18 +126,6 @@ export const getThrottleNetworkParam = function (argv) {
     return networkCondition
 }
 
-/**
- * validate throttleCpu param
- * @param  {Object}   argv cli params
- */
-export const getThrottleCpuParam = function (argv) {
-    const cpuRate = argv.throttleCpu || 4
```

Comment:
Wouldn't just removing the `|| 4` have fixed this issue?

With this PR, we move the validation to the protocol instead, which would make it take longer for the user to be told they entered an invalid argument for throttleCpu

Reply:
@yfangsl I fixed the conversion here: https://github.com/saucelabs/speedo/pull/52/files#diff-a7748843fbda5167d065231e3a2e7a1fR110